src/renderer/store/TimelineSpace/Modals/NewToot.ts

```diff
@@ -0,0 +1,249 @@
+  sensitive: false,
+  attachedMediaId: 0,
+  pinedHashtag: false,
+  hashtags: []
+})
+
+export const MUTATION_TYPES = {
+  CHANGE_MODAL: 'changeModal',
+  SET_REPLY_TO: 'setReplyTo',
+  UPDATE_INITIAL_STATUS: 'updateInitialStatus',
+  UPDATE_INITIAL_SPOILER: 'updateInitialSpoiler',
+  CHANGE_BLOCK_SUBMIT: 'changeBlockSubmit',
+  APPEND_ATTACHED_MEDIAS: 'appendAttachedMedias',
+  CLEAR_ATTACHED_MEDIAS: 'clearAttachedMedias',
+  REMOVE_MEDIA: 'removeMedia',
+  CHANGE_VISIBILITY_VALUE: 'changeVisibilityValue',
+  CHANGE_SENSITIVE: 'changeSensitive',
+  UPDATE_MEDIA_ID: 'updateMediaId',
+  CHANGE_PINED_HASHTAG: 'changePinedHashtag',
+  UPDATE_HASHTAGS: 'updateHashtags'
+}
+
+const mutations: MutationTree<NewTootState> = {
+  [MUTATION_TYPES.CHANGE_MODAL]: (state, value: boolean) => {
+    state.modalOpen = value
+  },
+  [MUTATION_TYPES.SET_REPLY_TO]: (state, message: Status) => {
+    state.replyToMessage = message
+  },
+  [MUTATION_TYPES.UPDATE_INITIAL_STATUS]: (state, status: string) => {
+    state.initialStatus = status
+  },
+  [MUTATION_TYPES.UPDATE_INITIAL_SPOILER]: (state, cw: string) => {
+    state.initialSpoiler = cw
+  },
+  [MUTATION_TYPES.CHANGE_BLOCK_SUBMIT]: (state, value: boolean) => {
+    state.blockSubmit = value
+  },
+  [MUTATION_TYPES.APPEND_ATTACHED_MEDIAS]: (state, media: Attachment) => {
+    state.attachedMedias = state.attachedMedias.concat([media])
+  },
+  [MUTATION_TYPES.CLEAR_ATTACHED_MEDIAS]: (state) => {
+    state.attachedMedias = []
+  },
+  [MUTATION_TYPES.REMOVE_MEDIA]: (state, media: Attachment) => {
+    state.attachedMedias = state.attachedMedias.filter(m => m.id !== media.id)
+  },
+  /**
+   * changeVisibilityValue
+   * Update visibility using direct value
+   * @param state vuex state object
+   * @param value visibility value
+   */
+  [MUTATION_TYPES.CHANGE_VISIBILITY_VALUE]: (state, value: number) => {
+    state.visibility = value
+  },
+  [MUTATION_TYPES.CHANGE_SENSITIVE]: (state, value: boolean) => {
+    state.sensitive = value
+  },
+  [MUTATION_TYPES.UPDATE_MEDIA_ID]: (state, value: number) => {
+    state.attachedMediaId = value
+  },
+  [MUTATION_TYPES.CHANGE_PINED_HASHTAG]: (state, value: boolean) => {
+    state.pinedHashtag = value
+  },
+  [MUTATION_TYPES.UPDATE_HASHTAGS]: (state, tags: Array<Tag>) => {
+    state.hashtags = tags
+  }
+}
+
+const actions: ActionTree<NewTootState, RootState> = {
+  updateMedia: async ({ rootState }, media: Attachment) => {
+    if (rootState.TimelineSpace.account.accessToken === undefined || rootState.TimelineSpace.account.accessToken === null) {
+      throw new AuthenticationError()
+    }
+    const client = new Mastodon(
+      rootState.TimelineSpace.account.accessToken,
+      rootState.TimelineSpace.account.baseURL + '/api/v1'
+    )
+    return Promise.all(
+      Object.keys(media).map(async id => {
+        return client.put<Attachment>(`/media/${id}`, { description: media[id] })
+      })).catch(err => {
+        console.error(err)
```

Comment:
__[ESLint]__ Error - Expected indentation of 6 spaces but found 8. (indent) ([link](https://sider.review/gh/repos/124234495/pulls/873#issue-3973717))

<sub>posted by Sider</sub>